.github/workflows/ci.yaml

```diff
@@ -709,6 +594,7 @@ jobs:
     if: always()
     needs:
       - lint-checks
+      - self-check
```

Comment:
I noticed that the slack notification can happen before self-check finishes. It turns out, this was missing.